app/src/main/res/values/strings.xml

```diff
@@ -73,7 +73,7 @@
   <string name="menu_nearby">Nearby</string>
   <string name="provider_contributions">My uploads</string>
   <string name="menu_share">Share</string>
-  <string name="menu_open_in_browser">View in Browser</string>
+  <string name="menu_view_file_page">View File Page</string>
```

Comment:
Could you please change the capitalization to `View file page`? Thanks!

Reply:
@nicolas-raoul done :)